net.veloren.veloren.yaml

```diff
@@ -34,8 +34,8 @@ modules:
 
       - type: archive
         only-arches: [aarch64]
-        url: https://static.rust-lang.org/dist/2023-04-20/rust-nightly-aarch64-unknown-linux-gnu.tar.xz
-        sha256: c82a0c4551513dd921d73b711166d0e999ad49e4b59ba090798855e724d75ae7
+        url: https://static.rust-lang.org/dist/2024-01-17/rust-nightly-aarch64-unknown-linux-gnu.tar.xz
+        sha256: 2b0af3d0a930e9356baf4252ea68d68e4ee7f4eebfb9c8376d06300567ab1149
         x-checker-data:
```

Comment:
Hi @xMAC94x, can you please remove the `x-checker-data` from here and below? Nightly modules should not be using x-checker and it's wasting Flathub's buildserver with a PR each week.